### PR TITLE
Introduction

### PR DESCRIPTION
**Pull request by perrygreenfield**
Split off from the original PR

### CODE REVIEW

File: 01-Introduction/01-Introduction.ipynb [hunk 1]
```diff
@@ -0,0 +1,193 @@
+{
```
Comment (WilliamJamieson) [inline 01-Introduction/01-Introduction.ipynb:1]: <p>The bullet nesting does not seem to be displaying correctly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (perrygreenfield) [reply 01-Introduction/01-Introduction.ipynb:1]: <p>I've redone the bulleting. But I'm not sure I addressed the issue. I agree that putting all sub-bullets under archive requirement was really mis-organized</p>

[the remark this answers is not in the document]
Comment (perrygreenfield) [reply 01-Introduction/01-Introduction.ipynb:1]: <p>I've redone the bulleting. But I'm not sure I addressed the issue. I agree that putting all sub-bullets under archive requirement was really mis-organized</p>

Comment (perrygreenfield) [reply 01-Introduction/01-Introduction.ipynb:1]: <p>I've reorganized the formatting, but if the issue persists (after my next commit) perhaps a screen shot illustrating the issue would help me.</p>

**Review by WilliamJamieson — DISMISSED** [review]

**Pull request merged**